src/main/kotlin/pet/project/app/service/BookService.kt

```diff
@@ -0,0 +1,44 @@
+package pet.project.app.service
+
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import pet.project.app.repository.BookRepository
+import pet.project.app.model.Book
+
+@Service
+class BookService(val bookRepository: BookRepository) {
+
+    fun create(book: Book) : Book {
+        return bookRepository.save(book)
+    }
+
+    fun getById(bookId: String): Book {
+        return bookRepository.findByIdOrNull(bookId) ?: throw Exception("No book with id = $bookId")
+    }
+
+    fun update(bookId: String, book: Book): Book {
+        if (!bookRepository.existsById(bookId)) {
+            throw Exception("No book with id = $bookId, to update")
+        }
+        book.id = bookId
+        return bookRepository.save(book)
```

Comment:
Prefer non-reversed conditions. It's easier to read
```suggestion
        return if (bookRepository.existsById(bookId)) {
            book.id = bookId
            bookRepository.save(book)
        } else {
            throw BookNotFoundException("No book with id = $bookId, to update")
        }
```

Reply:
Fixed: Investegated code conventions and updated code as suggested